ironic/resource_ironic_node_v1.go

```diff
@@ -585,3 +614,131 @@ func changePowerState(client *gophercloud.ServiceClient, d *schema.ResourceData,
 
 	return nil
 }
+
+// setRAIDConfig calls ironic's API to send request to change a Node's RAID config.
+func setRAIDConfig(client *gophercloud.ServiceClient, d *schema.ResourceData) (err error) {
+	var logicalDisks []nodes.LogicalDisk
+	var raid *metal3v1alpha1.RAIDConfig
+
+	raidConfig := d.Get("raid_config").(string)
```

Comment:
If this cast fails, whole function panicks. I think, for all castings in this PR, they should be handled to prevent possible panicks and gracefully return error. I'd prefer;

```
raidConfig, ok := d.Get("raid_config").(string)
if !ok{error}
```

Reply:
Or I'm missing something and we know all types 100% and zero probability for panick?

Reply:
[raid_config](https://github.com/openshift-metal3/terraform-provider-ironic/pull/53/commits/93fc8c6a7c83154da38ac9982331d795324ed313#diff-373c79495b73be3d62528c3a124f7bdf8966d21f21e2f8ab49dfc1754bf4eec9R187-R191) has been defined, so I think it is zero probability for panick